src/SearchButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function SearchButton() {
+  return (
+    <div>
+      <button className="btn btn-primary">Search IDs</button>
```

Comment:
How could we make this component more reusable? Right now it will always show "Search IDs" as the button text. Which isn't very useful if we wanted to use it somewhere else in the app. How could we change this so that it can be reused in multiple different places?